### PR TITLE
add buf config for protoc-gen-go-errors

### PR DESCRIPTION
**Pull request by akoserwal**
#### Description (what this PR does / why we need it):

* Is buf.build/go-kratos currently owned by go-kratos team?
* I raised PR for protoc-gen-go-errors, it can add for protoc-gen-go-http

### CODE REVIEW

File: cmd/protoc-gen-go-errors/buf.gen.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+version: v2
+plugins:
+  - remote: buf.build/protocolbuffers/go:v1.33.0
+    out: .
+    opt:
+      - paths=source_relative
```
Comment (shenqidebaozi) [inline cmd/protoc-gen-go-errors/buf.gen.yaml:6]: We need a blank line here

**Comment by akoserwal** [comment]
Require a Github action for pushing to buf registry, Generate a buf token which can be configured to the secrets

``` name: buf-push
on:
  push:
    branches:
      - main

jobs:
  build:
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v4
      - uses: bufbuild/buf-setup-action@v1
      - uses: bufbuild/buf-lint-action@v1
        with:
          input: "api"
      - uses: bufbuild/buf-breaking-action@v1
        with:
          input: "api"
          against: "https://github.com/${GITHUB_REPOSITORY}.git#branch=main,ref=HEAD~1"
      - uses: bufbuild/buf-push-action@v1
        with:
          input: "api"
          buf_token: ${{ secrets.BUF_TOKEN }}```

**Comment by shenqidebaozi** [comment]
> Require a Github action for pushing to buf registry, Generate a buf token which can be configured to the secrets

@akoserwal Do I have to complete this operation? Can you operate it

**Review by shenqidebaozi — APPROVED** [review]

**Comment by akoserwal** [comment]
> > Require a Github action for pushing to buf registry, Generate a buf token which can be configured to the secrets
> 
> @akoserwal Do I have to complete this operation? Can you operate it
> 
> We can discuss separately and merge the current PRs first

@shenqidebaozi 

- I would suggest creating an org/team on buf registry and generating a secret token

https://buf.build/docs/bsr/authentication#create-a-

- Which can be configured as `BUF_TOKEN` [github secrets ](https://docs.github.com/en/actions/security-for-github-actions/security-guides/using-secrets-in-github-actions)
- You can add the github actions with code pasted above

**Pull request merged**